output/components.sasslint.txt

```diff
@@ -1,6 +1,12 @@
 > sass-lint -v -q
 
 
+src/DateTimePickers/pickers/DatePicker/DatePicker.scss
+  38:11  error  A value of `0` is not allowed. `none` must be used  border-zero
+
+src/DateTimePickers/pickers/TimePicker/TimePicker.scss
+  17:9  error  Qualifying elements are not allowed for class selectors  no-qualifying-elements
```

Comment:
Sorry for this one, but this is needed, otherwise the theme has a better ponderation, and is applied.
The other ones are fixed